tools/arch-visualizer/viewer/src/components/HelpSystem.tsx

```
@@ -0,0 +1,299 @@
+            : "bg-white text-zinc-500 hover:bg-zinc-50 hover:text-zinc-800 border border-zinc-200"
+          }
+        `}
+        title="Help (?)"
+      >
+        ?
+      </button>
+
+      {/* Welcome guide overlay */}
+      {showWelcome && (
+        <div className="fixed inset-0 z-50 flex items-center justify-center p-4">
+          <div className="absolute inset-0 bg-black/60 backdrop-blur-sm" onClick={dismissWelcome} />
```

Biome (2.3.13)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)